src/HasteModuleLoader/__tests__/test_root/mapped_module_test.js

```diff
@@ -0,0 +1 @@
+module.exports = 'mapped_module';
```

Comment:
we have on internally and landing this diff without the header would fail. @zpao can we make this an eslint rule and put it into fbjs scripts?